### ISSUE TITLE
[NBug] Access to the path 'C:\Users\michal.kulinski\source\r-us\.g...

### ISSUE THREAD

**Issue by coola** [gitextensions/gitextensions#6908]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.UnauthorizedAccessException: Access to the path 'C:\Users\michal.kulinski\source\r-us\.git\COMMITMESSAGE' is denied.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at System.IO.Abstractions.FileWrapper.WriteAllText(String path, String contents, Encoding encoding)
   at GitCommands.CommitMessageManager.set_MergeOrCommitMessage(String value)
   at GitUI.CommandsDialogs.FormCommit.OnFormClosing(FormClosingEventArgs e)
   at System.Windows.Forms.Form.CheckCloseDialog(Boolean closingOnly)
```

## Environment
- Git Extensions 3.1.1.6049
- Build 2f872105b4eb42421d9fdafd68e90a9e27d20d96
- Git 2.21.0.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3416.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Did you have multiple instances of GE running?

**Comment by coola** [comment]
No, 
this is the case, when there was repo created with Admin rights. 
GE was run with ordinary user rights.
Instead of meaningful information it throws exception. 

Best regards
Michal Kulinski

**Comment by gerhardol** [comment]
I got this as well:

If COMMITMESSAGE cannot be read when opening FormCommit, NBug popup. That can be dismissed, but NBug appears when trying to close FormCommit too, the form does not close.

Not sure why I got this. There were no access to the file (according to ProcExp and resmon), I could not change access for the file.
No recollection of any specific done before this occur, the Symantec virus is a common cause for many issues.
(Have not checked the status after rebooting.)
